app/components/my-c.js

```diff
@@ -1,47 +0,0 @@
-import Ember from 'ember';
-
-const SCROLL_COUNTER = new Ember.Map();
-const SOME_STRING = 'HELLO';
-let LAST_COMPONENT;
-
-export default Ember.Component.extend({
-  scrolledComponents: [],
-  init() {
-    this._super(...arguments);
-    this.scrollCount = 0;
-    this.other();
-    SCROLL_COUNTER.set(this, { count: 0 });
-  },
-
-  other() {
-    this.foo = () => this;
-    window.foo = function() {
-      return SOME_STRING;
```

Comment:
@stefanpenner added this to cover this type of leak: https://github.com/trentmwillis/memory-leak-examples/blob/3d47e12f46964ab78f0f20c5e66c9cd8ce843e9f/app/utils/component-debug.js

Seem good?